backend/src/test/java/capstone/myfinancemanager/manager/controller/TransactionControllerIntegrationTest.java

```diff
@@ -77,4 +82,47 @@ void addTransaction() throws Exception {
                         """));
     }
 
+    @DirtiesContext
+    @Test
+    void deleteTransaction() throws Exception {
+
+        String saveResult = mockMvc.perform(post(
+                "/transactions")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content("""
+                           {
+                                    "userEmail": "a@a.com",
+                                    "description": "5",
+                                    "amount": 123.2,
+                                    "transactionDate": "1661866382913",
+                                    "category": "Essen",
+                                    "pictureId": "url",
+                                    "isIncome": true
+                                }
+                        """)
+        ).andReturn().getResponse().getContentAsString();
```

Comment:
nur als comment für zukunft, falls du deutsche umlaute in test data verwenden wirst, soll man StandardCharsets.UTF_8 verwenden.  Z.B.:
String actualStr = result.getResponse().getContentAsString(StandardCharsets.UTF_8);